docs/apache-airflow-providers-weaviate/operators/weaviate.rst

```diff
@@ -28,7 +28,7 @@ into the database.
 Using the Operator
 ^^^^^^^^^^^^^^^^^^
 
-The WeaviateIngestOperator requires the ``input_text`` as an input to the operator. Use the ``conn_id`` parameter to specify the Weaviate connection to use to
+The WeaviateIngestOperator requires the ``input_data`` as an input to the operator. Use the ``conn_id`` parameter to specify the Weaviate connection to use to
```

Comment:
Previously the operator had `input_json` and now `input_data`. Does this count as breaking compatibility? And the doc had `input_text` and now `input_data`. Was the old content simply wrong? It may be worthwhile to dig out some history on why the doc and code did not match.

Reply:
Yes, that's true. I'll add a deprecation warning for `input_json` and introduce `input_data` while supporting both parameters. Thanks for pointing this out.